main.go

```diff
@@ -137,3 +143,40 @@ func run(kubeConfig string) error {
 
 	return nil
 }
+
+func enableKubeVirtFeatureGateHostDevices() {
```

Comment:
the `enableKubeVirtFeatureGateHostDevices` should have kind of return value, if `error` is not acceptable, then return error will let POD crash, so as to retry;  if `error` is acceptable, use a `flag` / controller variable  to remark it, and skip the related feature later

silently skip `error` is not, because the logrus log maybe truncated after long running, it's hard to trouble-shooting.